app/views/portal/client_logins/link_sent.html.erb

```diff
@@ -1,9 +1,22 @@
 <% content_for :page_title, t(".title") %>
 
 <% content_for :card do %>
+  <div class="question__illustration spacing-above-0">
```

Comment:
question__illustration class should already have top margin of 0, so i dont know if the spacing-above-0 is necessary here.

Reply:
Good call. I tested this, and it's not required, so I removed it in a push straight to main: https://github.com/codeforamerica/vita-min/commit/f52efddaebc8387aeec383ed341ae367c59a467a